crates/python_extractor/src/lib.rs

```diff
@@ -120,68 +121,90 @@ pub fn read_file_to_str(input_file: &PathBuf) -> Result<String, anyhow::Error> {
     }
 }
 
-pub async fn extract_python(
-    relative_input_paths: String,
-    working_directory: PathBuf,
-    output: PathBuf,
-    label_or_repo_path: String,
+async fn extract_file(
```

Comment:
write a function that just runs on a single file to reduce the size of the function below that calls it.